spec/requests/vendor_api/get_multiple_applications_spec.rb

```diff
@@ -2,26 +2,37 @@
 
 RSpec.describe 'Vendor API - GET /api/v1/applications', type: :request do
   include VendorApiSpecHelpers
+  include FactoryHelpers
 
-  it 'returns applications of the authenticating provider' do
+  it 'returns applications scoped by `provider`' do
```

Comment:
I think this is a regression — the provider to fetch applications for is now inferred from `currently_authenticated_provider`, so the original title should stand and `UCL` should not be specified in the URL (the spec continues to pass if it's removed)